packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -876,15 +882,8 @@ String _hex(List<int> bytes) {
 }
 
 String _calculateSha(File file) {
-  final Stopwatch sw = Stopwatch()..start();
   final List<int> bytes = file.readAsBytesSync();
-  globals.printTrace('calculateSha: reading file took ${sw.elapsedMilliseconds}us');
```

Comment:
I reviewed the data here and basically this is always super fast, the reported issue was likely unrelated